### ISSUE TITLE
Add api for sanitized config

### ISSUE THREAD

**Issue by LostKobrakai** [nerves-networking/vintage_net#199]
I'm showing vintage_net information in a custom web interface, but currently sanitization of interface config is only done for `VintageNet.info`, but unavailable as an API.

**Comment by fhunleth** [comment]
Agree. It seems reasonable to expose the function that sanitizes configuration maps so that you can programmatically use it too.
